### DISCUSSION TITLE
can't flash my Bigtreetech TFT3.5 V3 E3

### DISCUSSION THREAD

**Discussion by rmaltais** [teeminus/NoTouchScreenFirmware#68] [General]
Hi,
i can't flash my screen firmware with the NoTouchScreenFirmware here.  I have always same result error "Illegal FLASH APP!"

BIGTREE_TFT35_V3.0_E3.26.x

Insteed, i can flash with original one by BigtreeTech.

Anyone can fix this issue ?

**Comment by teeminus** [comment]
Try compiling the firmware yourself.

**Reply by rmaltais** [reply]
Gotcha! Work now.

Thank for all.
